health/notifications/alerta/README.md

```diff
@@ -84,11 +72,11 @@ We can test alarms using the standard approach:
 /opt/netdata/netdata-plugins/plugins.d/alarm-notify.sh test
```

Comment:
`/opt/netdata/` will be there only if the user uses a static binary install. Sometimes it can be `/usr/....` etc. Not sure how we deal with paths in other parts of the documentation...

Reply:
> /opt/netdata/**netdata-plugins**/plugins.d/alarm-notify.sh test
> **netdata-plugins**

First time I see **netdata-plugins** directory 😄 🤔

Reply:
Oh, it is [static build](https://github.com/netdata/netdata/blob/22cdd71590f7214a6432b6aa89be9dd07abdbb7f/packaging/makeself/install-or-update.sh#L196). Then it can't be `/usr/.... `.

Reply:
Yes, but if we put a path in a document, we somehow need to cover all (most?) of the possible situations. (Of course besides the user having specified a completely different path). I.e. the `/opt/netdata/netdata-plugins/plugins.d/alarm-notify.sh` won't work for from source installations.

Reply:
Anyway, I think we don't need to fix this in this PR.

Reply:
Ok, should we add a note, something like "depending on the way Netdata was installed, the `alarm-notify.sh` script might be...", or is it too much? I'm ok if we leave it like that, we can pick it up at some other point...

Reply:
>  I'm ok if we leave it like that, we can pick it up at some other point...
👆 

My point is this problem exists for all health readmes ([Discord](https://github.com/netdata/netdata/tree/master/health/notifications/discord) for example) and many other docs.

Reply:
Agree. ok!